bndtools.core/src/bndtools/central/RefreshFileJob.java

```diff
@@ -14,42 +17,64 @@
 public class RefreshFileJob extends WorkspaceJob {
 
     private final boolean derived;
+    private final List<File> files;
 
-    private final IResource resource;
-    private final int depth;
+    private IResource resource;
+    private int depth;
 
     public RefreshFileJob(File file, boolean derived) throws Exception {
         super("Refreshing " + file);
         this.derived = derived;
+        this.files = new ArrayList<File>();
+        this.files.add(file);
```

Comment:
You should use Collections.singletonList(file) here instead of creating an array list and then adding the one file.